### ISSUE TITLE
Swiper aria-labels are not translated

### ISSUE THREAD

**Issue by emilyberghen** [statikbe/craft#528]
Swiper A11Y strings are not translated so not accessible e.g. the navigation buttons. Translations added in SHADNS after Anysurfer audit.

**Comment by KarelJanVanHaute** [comment]
@emilyberghen Weet jij nog waar dit over ging? Neem jij dit verder op?
